src/encoded/static/components/item-pages/HiGlassViewConfigView.js

```diff
@@ -20,20 +20,19 @@ import { CollapsibleItemViewButtonToolbar } from './components/CollapsibleItemVi
 import { Wrapper as ItemHeaderWrapper, TopRow, MiddleRow, BottomRow } from './components/ItemHeader';
 import { EmbeddedItemSearchTable } from './components/tables/ItemPageTable';
 import DefaultItemView from './DefaultItemView';
-
-
+import { EditableField, FieldSet } from '@hms-dbmi-bgm/shared-portal-components/es/components/forms/components/EditableField';
 
 export default class HiGlassViewConfigView extends DefaultItemView {
 
-    itemHeader(){
+    itemHeader() {
         const itemActionsDescriptions = {
-            'edit' : 'Edit the properties of this Item.',
+            'edit': 'Edit the properties of this Item.',
         };
 
         return (
             <ItemHeaderWrapper {..._.pick(this.props, 'context', 'href', 'schemas', 'windowWidth')}>
                 <TopRow typeInfo={this.typeInfo()} itemActionsDescriptions={itemActionsDescriptions} />
-                <MiddleRow />
+                <MiddleRow isInlineEditable={true} />
```

Comment:
(Not Important) Can also just have `<MiddleRow isInlineEditable />` which is analogous to `<MiddleRow isInlineEditable={true} />`